.github/workflows/workflow-linter.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: Checkout Branch
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
-          repository: bitwarden/gh-actions
+          repository: ${{ github.repository }}
```

Comment:
```suggestion
```

`${{ github.repository }}` is the default value, so we can safely delete this line.

Reply:
Doc: https://github.com/actions/checkout?tab=readme-ov-file#usage